ui/resizable.js

```diff
@@ -199,6 +204,10 @@ $.widget("ui.resizable", $.ui.mouse, {
 				if (this.handles[i].constructor === String) {
 					this.handles[i] = this.element.children( this.handles[ i ] ).first().show();
 				}
+				else if (this.handles[i] && (this.handles[i].jquery || this.handles[i].nodeType)) {
```

Comment:
The first check here `this.handles[i].jquery` is unnecessary. The previous check is already looking at `this.handles[i].constructor`.